ystafdb/foaf.py

```diff
@@ -7,10 +7,10 @@
 from .config_parser import get_config_data
 
 
-def generate_foaf_uris(output_base_dir):
+def generate_foaf_uris(args):
     """Note the URIs needed for units. They all come from the Ontology of Units
     of Measure."""
-    output_base_dir = Path(output_base_dir)
+    output_base_dir = Path(args.outdir)
```

Comment:
Is outdir directly from the input `args`?  What if the dir does not exits? should be checked, is it?

Reply:
If -o argument does not exist, it defaults to *output*.
If it does not exist, it will be created automatically.